.github/workflows/ci.yaml

```diff
@@ -168,6 +168,161 @@ jobs:
       docker_repo_username: ${{ secrets.QUAY_USERNAME }}
       docker_repo_password: ${{ secrets.QUAY_PASSWORD }}
 
+  push-rhcc:
```

Comment:
split these jobs into a separate workflow that is only triggered [on push of new tag](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs)

Reply:
since the logic for rhcc, gcr and dockerhub is mostly the same did you try running this using a [job matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#example-expanding-configurations)?
could pass in registry, registry username, registry password and reuse the same steps

Reply:
done